scripts-circleci/github-package-helm-dispatch.sh

```diff
@@ -18,6 +18,12 @@ if [ -z "${VERSION}" ]; then
   exit 1
 fi
 
+# If VERSION is SEMVER set TAG_REF for workflow_dispatch
+if echo "${VERSION#v}" | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$'; then
+    echo "TAG_REF=v${VERSION#v}"
+    TAG_REF="v${VERSION#v}"
+fi
```

Comment:
When cloud releaser used there is no TAG trigger so check if the version is semver then use that as the REF